stdlib/int.mli

```diff
@@ -66,11 +66,11 @@ val abs : int -> int
 
 val max_int : int
 (** [max_int] is the greatest representable integer,
-    [2{^[Sys.int_size - 1]} - 1]. *)
+    [2]{^[Sys.int_size - 1]}[ - 1]. *)
```

Comment:
This introduces small gaps between the elements with Odoc and the superscript seems weird with the pseudo ocaml syntax that is used. What about this:
```
[2 ^ (Sys.int_size - 1) - 1]
```

As a reader, I can easily pretend that `^` doesn't mean string concatenation, however I have a hard time reading the unusual superscript.

Reply:
If you remove the spaces it looks fine in my opinion. That is:
```
[2]{^[Sys.int_size-1]}[-1].
```
yields
<img width="449" alt="Screenshot 2023-04-28 at 18 25 46" src="https://user-images.githubusercontent.com/485596/235202321-9a299518-f2bf-4a42-ac39-aae6d985d7dc.png">